public/css/main.css

```diff
@@ -20,14 +19,96 @@ a {
 }
 
 /*** GENERAL ***/
-
 body {
   background-color: #F2F2F2;
 }
 
 /*** HEADER BAR ***/
+.header {
+  background-color: #409A93;
+  font-family: 'Noto Sans', sans-serif;
+  padding: 2.5rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.naz-events{
+  color: white;
+  font-weight: bold;
+  font-size: 2.3rem;
+}
+
+.calendar {
+  font-size: 2.3rem;
+  color: white;
+  margin-top: 0.1rem;
+}
+
+.plus{
+  margin-top: 0.2rem;
+  font-size: 2.3rem;
+  color: white;
+}
+
+.languageNav {
+  background: #F2F2F2;
+  width: 100%;
+  padding: 0.6rem;
+  box-shadow: 0.4rem 0.5rem 0.5rem #DBDBDB;
+}
+
+.langBtn{
+  width: 48.5%;
+  font-weight: bolder;
+  font-size: 1.5rem;
+  color: #5C5C5C;
+  padding: 0.5rem;
+  background: #DBDBDB;
+  border: 0.1rem solid #DBDBDB;
+  border-radius: 0.1rem;
+  font-family: 'Lato', sans-serif;
```

Comment:
Not essential, but for future reference you can put all the font details in one property: 

`font: bolder 1.5rem 'Lato', sans-serif`